modules/fscloud/variables.tf

```diff
@@ -65,3 +65,43 @@ variable "cbr_rules" {
   description = "(Optional, list) List of CBR rules to create."
   default     = []
 }
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "cos_destination_name" {
+  type        = string
+  description = "The name to give the IBM Cloud Object Storage destination which will be created for storage of failed delivery events."
+  default     = "COS Destination"
+}
+
+variable "cos_bucket_name" {
+  type        = string
+  description = "The existing bucket name in IBM cloud object storage instance."
```

Comment:
The name of an existing IBM Cloud Object Storage bucket which will be used for storage of failed delivery events.